<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QRplusplusClass</class>
 <widget class="QMainWindow" name="QRplusplusClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>QRplusplus</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupInput">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>760</width>
      <height>360</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>760</width>
      <height>360</height>
     </size>
    </property>
    <property name="title">
     <string>Input</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Text:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>700</width>
       <height>280</height>
      </rect>
     </property>
     <property name="text">
      <string>Insert text here...</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupCustomize">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>400</y>
      <width>600</width>
      <height>120</height>
     </rect>
    </property>
    <property name="title">
     <string>Customize</string>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Pixel Size:</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_pixelsize">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>121</width>
       <height>22</height>
      </rect>
     </property>
     <property name="value">
      <number>5</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>30</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Error Correction:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_errorcor">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>60</y>
       <width>121</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Low</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Medium</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Quartile</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>High</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>30</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Margin Size:</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_marginsize">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>60</y>
       <width>121</width>
       <height>22</height>
      </rect>
     </property>
     <property name="value">
      <number>4</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>30</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Color:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_color">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>60</y>
       <width>121</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Black</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QPushButton" name="generateButton">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>410</y>
      <width>141</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Generate</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exitButton">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>490</y>
      <width>141</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="qrplusplus.qrc"/>
 </resources>
 <connections/>
</ui>
